### PR TITLE
feat: Dis 21 migrazione dominio nodo v3

### PR DESCRIPTION
**Pull request by re-sh-cloud-arch**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/nodo-secret/storageaccount.tf [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  versioning_name            = "versioning"
-  enable_versioning          = false
-  resource_group_name        = azurerm_resource_group.sec_rg.name
-  location                   = var.location
-  advanced_threat_protection = false
-  allow_blob_public_access   = false
+  name                     = replace("${local.product}-${var.domain}-nodocerts-sa", "-", "") # nodocerts<dev|uat|prod>
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
```
Comment (diegolagospagopa) [inline src/domains/nodo-secret/storageaccount.tf:7]: is correct that must be "LRS" and not GRS o something more?

File: src/domains/nodo-secret/99_locals.tf [hunk 2]
```diff
@@ -4,4 +4,6 @@ locals {
 
   monitor_action_group_slack_name = "SlackPagoPA"
   monitor_action_group_email_name = "PagoPA"
+
+  pipe_service_principel_application_id = "b7375314-3eff-4760-9952-84f96ee669cf"
```
Comment (diegolagospagopa) [inline src/domains/nodo-secret/99_locals.tf:8]: can you put a comment with the name of the SP, to simplify the search pleasee

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: src/domains/nodo-secret/storageaccount.tf (hunk 1 above)
Comment (re-sh-cloud-arch) [reply src/domains/nodo-secret/storageaccount.tf:7]: at the moment is correct. with DR implementation will be migrate to GZRS

File: src/domains/nodo-secret/99_locals.tf (hunk 2 above)
Comment (re-sh-cloud-arch) [reply src/domains/nodo-secret/99_locals.tf:8]: with application ID in Azure Portal search is more quickly then with name. use name of SP for search give more than one result

**Review by fmlisco — APPROVED** [review]

**Review by uolter — APPROVED** [review]

**Pull request merged**